src/EditorFeatures/CSharpTest/Classification/SemanticClassifierTests.cs

```diff
@@ -3903,5 +3903,65 @@ public async Task NotNativeUnsignedInteger()
                 code: @"nuint i = 0;",
                 expected: Classifications(Class("nuint")));
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Classification)]
+        public async Task StaticBoldingMethodName()
+        {
+            await TestAsync(
+@"class C
+{
+    public static void Method()
+    {
+        System.Action action = Method;
+    }
+}",
+            Namespace("System"),
+            Delegate("Action"),
+            Method("Method"),
+            Static("Method"));
```

Comment:
i'm not really understanding why this wouldn't have worked before.  there was only one 'Method', and it's static, so why do we need to care about multiple symbols?

Reply:
This case came up as a "does it work?" so I believe @akhera99 added the test both to confirm we don't break it and also just to make sure it's working too.